simulator/interfaces.py

```
@@ -0,0 +1,82 @@
+    @property
+    @abstractmethod
+    def mqtt_topic(self) -> str:
+        """MQTT topic for this object"""
+
+    @abstractmethod
+    def on_message(self, mqtt_client, userdata, msg):
+        """Handler for mqtt_topic"""
+
+    @property
+    def loggername(self) -> str:
+        """Name of the logger"""
```

Ruff (0.11.9)

`AirportComponent.loggername` is an empty method in an abstract base class, but has no abstract decorator

(B027)